src/main/java/com/google/sps/Constants.java

```diff
@@ -19,5 +19,12 @@
   public static final String BUSINESS_PICTURES = "pictures";
   public static final String BUSINESS_RATING = "rating";
   public static final String BUSINESS_REVIEWS = "reviews";
+  public static final String BUSINESS_KEY = "businessID";
```

Comment:
do you mind changing this to 

public static final String BUSINESS_ID = "businessID";

I'm using this specificly for my edit.html form so i'm afraid when this PR gets merged in it will overwrite my changes and break it

Reply:
Done!